build-parent/pom.xml

```diff
@@ -381,7 +381,7 @@
             <dependency>
                 <groupId>me.escoffier.certs</groupId>
                 <artifactId>certificate-generator-junit5</artifactId>
-                <version>0.5.0</version>
+                <version>0.5.1-SNAPSHOT</version>
```

Comment:
Mmmmh, we don't want that in `main` :).

Reply:
Damn... I forgot to rollback this. Anyway it does not work :-D